patches/react-native+0.70.4-alpha.2.patch

```diff
@@ -0,0 +1,172 @@
+diff --git a/node_modules/react-native/Libraries/Components/TextInput/RCTTextInputViewConfig.js b/node_modules/react-native/Libraries/Components/TextInput/RCTTextInputViewConfig.js
```

Comment:
As mentioned earlier, we should remove this.

Reply:
This will be removed with the RN update